src/pages/settings/Subscription/SubscriptionPlan/SubscriptionPlanCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {SvgProps} from 'react-native-svg';
+import type {ValueOf} from 'type-fest';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import {PressableWithFeedback} from '@components/Pressable';
+import SelectCircle from '@components/SelectCircle';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import type CONST from '@src/CONST';
+
+type PersonalPolicyTypeExludedProps = Exclude<ValueOf<typeof CONST.POLICY.TYPE>, 'personal'>;
+
+type SubscriptionPlanCardProps = {
+    index: number;
+    plan: {
+        title: string;
+        src: React.FC<SvgProps>;
+        benefits: string[];
+        description: string;
+        isSelected: boolean;
+        type: PersonalPolicyTypeExludedProps;
+    };
+
+    onPress: (type: PersonalPolicyTypeExludedProps) => void;
+};
+function SubscriptionPlanCard({plan, index, onPress}: SubscriptionPlanCardProps) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    return (
+        <View style={[styles.borderedContentCard, styles.flex1, styles.mt5, styles.p5, index === 0 && styles.mr3]}>
```

Comment:
```suggestion
        <View style={[styles.borderedContentCard, styles.flex1, styles.mt5, styles.p5, index === 0 && styles.mr3, plan.isSelected && styles.borderColorFocus]}>
```

Let's add green border on the selected option